spec.html

```diff
@@ -31044,7 +31044,7 @@
             If _obj_ is an exotic object (including Proxy exotic objects) whose behaviour for [[Get]], [[Set]], [[Delete]], and [[GetOwnProperty]] is not the ordinary object implementation of these internal methods.
           </li>
           <li>
-            If any index index property of _obj_ whose name is a nonnegative integer less than _len_ is an accessor property or is a data property whose [[Writable]] attribute is *false*.
+            If any index property of _obj_ whose name is a nonnegative integer less than _len_ is an accessor property or is a data property whose [[Writable]] attribute is *false*.
```

Comment:
Wolfram and Google's definitions use "nonnegative" as the canonical term. Wikipedia and Wiktionary seem to prefer "non-negative". I guess both terms are accepted, so there's no need to fix what is not broken.